LayoutTests/imported/w3c/web-platform-tests/pointerlock/pointerlock_unadjustedMovement.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+</head>
+<body>
+    <h2>Description</h2>
+    <p>This test validates that pointer lock accepts the unadjustedMovement option.</p>
+    <hr/>
+
+    <button id="Button">lockTarget</button>
+    <div id="target">Target</div>
+
+    <script type="text/javascript" >
+        let button = document.getElementById('Button');
+        let target = document.getElementById('target');
+
+        async_test(t => {
+            button.addEventListener('mousedown', t.step_func(() => {
+                const p = target.requestPointerLock({unadjustedMovement: true});
+                p.then(t.step_func(() => {
+                    assert_equals(document.pointerLockElement, target);
+                    t.done();
+                })).catch(t.step_func((error) => {
+                    // requestPointerLock may throw NotSupportedError to say unadjustedMovement isn't supported.
+                    assert_throws_dom("NotSupportedError", () => { throw error; });
+                    // but to pass this test fully, it unadjustedMovement must be supported.
+                    assert_unreached("unadjustedMovement must be implemented.");
```

Comment:
I don't understand this check, tbh.

Reply:
From the [proposed spec](https://github.com/w3c/pointerlock/pull/49/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R499):

> If options["unadjustedMovement"] is true and the platform does not support unadjustedMovement:
> ... Reject promise with a "NotSupportedError" DOMException.

So the assert_unreached check is saying that unadjustedMovement must be supported in order to pass this test. The previous assert_throws_dom check is a debugging aid. If an implementation hit that error then they have something else wrong beyond simply not implementing unadjustedMovement.